configure

```diff
@@ -950,8 +948,8 @@ Optional Libraries:
 
   --with-opengl-prefix=DIR      prefix where OpenGL (ES) is installed (optional)
   --disable-opengl-game         disable OpenGL (ES) support in game [autodetect]
-  --disable-opengl-game-shaders disable use of modern OpenGL with shaders
-  --force-opengl-game-es2       only detect OpenGL ES2
+  --disable-opengl-game-classic disable OpenGL (ES) classic support in game
```

Comment:
"OpenGL (ES) classic" I think it should be "OpenGL classic". on another thought potentially we can support gles1, but we are not ready yet.

Reply:
ah yes, you are right, that was bad copy paste